votes/0572-tammyxia-as-migration-committer.md

```diff
@@ -0,0 +1,24 @@
+# A Vote for Tammyxia as Migration Committer
+
+## Proposal
+
+[@Tammyxia](https://github.com/Tammyxia) has been working on quality improvement of `pingcap/ticdc` for several months, she has found many bugs from TiCDC and made great contributions to TiCDC tests evolution.
+
+* [Authored bug issues](https://github.com/pingcap/ticdc/issues?q=is%3Aissue+author%3ATammyxia+label%3Atype%2Fbug+).
+
+I (@amyangfei) hereby nominate @Tammyxia as Migration Committer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team Migration
+
+## Result
+
+Approved by 1 binding votes, 1 non-binding vote:
+
+- lance6716 (binding)
```

Comment:
```suggestion

- amyangfei (binding)
- lance6716 (binding)
```

@amyangfei please consider count yourself a binding vote and conclude the result.

Reply:
updated

Reply:
@amyangfei for "conclude the result", it means `/unhold` and `/merge` the PR 😄